README.md

```diff
@@ -124,12 +124,39 @@ request.applePayDisabled = !canMakePayments
 
 ### 3D-Secure + Drop-In
 
-The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, then just enable it in the BTDropInRequest and set an amount.
+The new Drop-In supports 3D-Secure verification. If you have enabled 3D-Secure in the control panel, then just enable it in the BTDropInRequest and set an amount. Then, create a BTThreeDSecureRequest() object and set as many fields on it as possible; the more fields that are set, the less likely it is for a user to be presented with a challenge. Make sure to attach this object to the BTDropInRequest before use.
 
 ```swift
-let request =  BTDropInRequest()
+let request = BTDropInRequest()
 request.threeDSecureVerification = true
 request.amount = "1.00"
+
+let threeDSecureRequest = BTThreeDSecureRequest()
+threeDSecureRequest.threeDSecureRequestDelegate = self
+
+threeDSecureRequest.amount = 1.0
```

Comment:
The amount doesn't need to be two 0's for decimal places?

Reply:
It's just a float, so that shouldn't matter. We could change the README to 1.00 so it's clearer if you'd prefer, but it's functionally equivalent.

Reply:
Shouldn't this be a string when it's sent to the GW?

Reply:
From what I could find in the docs and the demo app, it looks like BTDropInRequest.amount is a string and BTThreeDSecureRequest.amount is an NSDecimalNumber.

Reply:
[Confirming](https://github.braintreeps.com/team-sdk/braintree-ios/blob/214c01b8db5fcb965f59ff3f7be56f1536ecf2ef/Demo/Features/3D%20Secure/BraintreeDemoThreeDSecurePaymentFlowViewController.m#L101) it is a `NSDecimalNumber`. Should could write something like ...

``` 
request.amount = [NSDecimalNumber decimalNumberWithString:@"10.32"];
``` 
to be the most clear of casting a string to a number.

Reply:
We could write `request.amount = NSDecimalNumber(string: "10.00")`, but I feel like that might be even more confusing than just changing this to 10.00.

Reply:
I don't feel too strongly either way. But whichever we use, number or string, should show two decimal places.

Reply:
Definitely agree with @scannillo

Reply:
Updated! Would love 👀so we can get this merged.

Reply:
So, I'm always confused about `request.amount` vs. `threeDSecureRequest.amount`, and I have a feeling merchants will be too. My understanding is that we want merchants to use `threeDSecureRequest.amount`, not `request.amount`. So, maybe we should just remove `request.amount` from this code snippet?

I know it's not part of this PR, but shouldn't we also mark `request.amount` as deprecated?